.teamcity/performance-test-durations.json

```diff
@@ -14,13 +14,13 @@
         "linux" : 1775000
     } ]
 }, {
-  "scenario" : "org.gradle.performance.regression.android.RealLifeAndroidBuildPerformanceTest.abi change",
+  "scenario" : "org.gradle.performance.regression.android.AndroidIncrementalExecutionPerformanceTest.abi change",
```

Comment:
The process around this JSON file is unclear to me. Does someone need to remember to run something to update their contents? Is there any manual step here to take? What happens if updating the JSON is forgotten?

If we can generate the JSON's contents automatically, why check it in, and not generate it during every build in a task (and use the build cache to avoid duplicate work)?

Reply:
If you don't update the JSON file, then the generation of the Teamcity configuration will fail. We have a build which checks this. You can update the times from the performance test database, if there is already some data for the performance test. Since we never did run the test on `master` (I changed the class name), there is no data. So I need to update the file manually.